sleap/gui/app.py

```diff
@@ -1364,15 +1364,15 @@ def updateStatusMessage(self, message: Optional[str] = None):
                     )
                     message += " in video"
 
-            lf = self.state["labeled_frame"]
-            n_instances = 0 if lf is None else len(lf)
+            visible_instances = self.state["labeled_frame"].instances_to_show
+            n_instances = 0 if visible_instances is None else len(visible_instances)
```

Comment:
I don't think `LabeledFrame.visible_instances` ever returns `None`? Again, if `self.state["labeled_frame"] == None`, you won't get a `None` back from an attribute that it doesn't have.

Reply:
Updated this to one again check if self.state['labeled_frame'] is None, but still use LabeledFrame.instances_to_show. However, I added a note to revisit this line if either the code for LabeledFrame.instances_to_show or LabeledFrame.unused_predictions changes.